routes/apply_discount.js

```diff
@@ -232,6 +234,59 @@ const responseSchema = {
   'type': 'object',
   'additionalProperties': false,
   'properties': {
+    'buy_together': {
+      'type': 'array',
+      'maxItems': 300,
+      'items': {
+        'type': 'object',
+        'required': [ 'discount' ],
+        'additionalProperties': false,
+        'properties': {
+          'product': {
+            'type': 'object',
+            'maxProperties': 100,
+            'patternProperties': {
+              '^([a-f0-9]{24})$': {
+                'type': 'object',
+                'properties': {
+                  'quantity': {
+                    'type': 'number',
+                    'minimum': 0,
+                    'maximum': 9999999,
+                    'description': 'Product quantity'
+                  },
+                },
+                'required': ['quantity']
```

Comment:
Então, parecido com isso aqui de novo https://github.com/ecomplus/modules-api/pull/17#discussion_r925034392 😛 
Não tem porque você forçar um array de objetos com uma propriedade só, dá pra ser só:
```js
products: [{
  p1: 1,
  p2: 20,
}]
```

Reply:
Não precisa de:
```js
[{
  p1: { quantity: 1 },
  p2: { quantity: 20 },
}]
```